### ISSUE TITLE
[UAC - Set password] Input field is missing required

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5881]
## Quick Summary
<img width="485" alt="Screenshot 2023-05-26 at 15 17 01" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/4bf1b402-e3d0-4d06-970c-e5efd656efe0">

`/uac/set-password`
- The `Password` and `Confirm password` fields are required but `required` attribute is missing.

`/uac/set-recovery-question`
- The `Security question` and `Answer` fields are required but `required` attribute is missing.

`/uac/mfa-select`
- `Add a second layer of security to protect your account.` field is required  but `required` attribute is missing.

`/uac/mfa-okta/verify
- `One-time security code`  field is required  but `required` attribute is missing.

`/uac/mfa-google-auth/verify`
- `One-time security code`  field is required  but `required` attribute is missing.

`/uac/mfa-phone/verify`
- `One-time security code`  field is required  but `required` attribute is missing.

`/uac/mfa-sms`
- `phone number` is required

`/uac/mfa-sms/verify`
- `One-time security code`  field is required  but `required` attribute is missing.

`/uac/mfa-phone`
- `phone number` is required

`/uac/mfa-phone/verify`
- `One-time security code`  field is required  but `required` attribute is missing.

`/uac/mfa-email/verify`
- `One-time security code`  field is required  but `required` attribute is missing.

## Accessibility Violation
[Deque Link](https://docs.deque.com/issue-help/1.0.0/en/aria-required-missing
)

## Action Requested

Add `required` attribute
Add `*` to the label text

## Source Code (optional)

[which file or line is the issue attached to?]: #
